### ISSUE TITLE
Consider `safebrowsing-enable-enhanced-protection` as default browser flag

### ISSUE THREAD

**Issue by ka3de** [grafana/xk6-browser#773]
After the [removal](https://github.com/grafana/xk6-browser/issues/771) of the flag `disable-client-side-phishing-detection` as default browser flag for being deprecated, we should consider setting `safebrowsing-enable-enhanced-protection` by default, as this flag is mentioned in the chromium project [issue](https://bugs.chromium.org/p/chromium/issues/detail?id=360220) that deprecates the mentioned flag, as a possible replacement for it.

Related: #771.

**Comment by ankur22** [comment]
An observation I've made while working on Dockerizing a browser is:

1. I can't seem to run a browser under a new user in a Docker container, which leads to
2. If running a browser as `root`, the browser will not start under `root` unless `--no-sandbox` is added.

My assumption then is that `--no-sandbox` has been added as a default to overcome issues where the browser was being started under `root`. I tried running a browser on my local machine as my own user (not `root`) and I was able to to start it without `--no-sandbox`.

**Comment by inancgumus** [comment]
@ankur22 

Yes, I've noticed Playwright does something similar. See [this](https://playwright.dev/docs/docker) for more information.

> By default, the Docker image will use the root user to run the browsers. This will disable the Chromium sandbox which is not available with root. If you run trusted code (e.g. End-to-end tests) and want to avoid the hassle of managing separate user then the root user may be fine. For web scraping or crawling, we recommend to create a separate user inside the Docker container and use the seccomp profile.
